sencha-workspace/SlateCSV/app/view/Importer.js

```diff
@@ -1,9 +1,11 @@
-/*jslint browser: true, undef: true, laxcomma:true *//*global Ext*/
+/*jslint browser: true, undef: true, plusplus: true */ /*global Ext*/
```

Comment:
to avoid 
app/view/Importer.js|229| jshint warning: Unexpected use of '++'.
app/view/Importer.js|246| jshint warning: Unexpected use of '++'.

I intend on refactoring those lines, but for now it avoids the popup